src/components/pagination/pagination.js

```diff
@@ -0,0 +1,150 @@
+import '@brightspace-ui/core/components/button/button-icon.js';
+import '@brightspace-ui/core/components/inputs/input-number.js';
+import '@brightspace-ui/core/components/inputs/input-text.js';
+import { css, html, LitElement } from 'lit';
+import { LocalizeLabsElement } from '../localize-labs-element.js';
+import { RtlMixin } from '@brightspace-ui/core/mixins/rtl-mixin.js';
+import { selectStyles } from '@brightspace-ui/core/components/inputs/input-select-styles.js';
+
+class Pagination extends RtlMixin(LocalizeLabsElement(LitElement)) {
+
+	static get properties() {
+		return {
+			pageNumber: { type: Number, attribute: 'page-number', reflect: true },
+			maxPageNumber: { type: Number, attribute: 'max-page-number', reflect: true },
+			showItemCountSelect : { type: Boolean, attribute: 'show-item-count-select', reflect: true },
+			itemCountOptions : { type: Array, attribute: 'item-count-options' },
```

Comment:
Does using the `item-count-options` attribute actually work? I think we'd need to add a custom converter, no? Maybe just not worth having the attribute at all.